ocean_provider/user_nonce.py

```diff
@@ -1,79 +1,29 @@
 import logging
 
-from ocean_utils.data_store.storage_base import StorageBase
+from ocean_provider.myapp import UserNonce, db
 
 logger = logging.getLogger(__name__)
 
 
-class UserNonce:
-    FIRST_NONCE = 0
+def get_nonce(address):
+    result = UserNonce.query.filter_by(address=address).first()
 
-    def __init__(self, storage_path=None):
-        self._storage_path = storage_path
-        NonceStorage(storage_path, create_table=True)
+    return result.nonce if result else UserNonce.FIRST_NONCE
 
-    @property
-    def storage(self):
-        return NonceStorage(self._storage_path, create_table=True)
 
-    def get_nonce(self, address):
-        nonce = self.storage.read_nonce(address)
-        if nonce is not None:
-            return int(nonce)
-        return UserNonce.FIRST_NONCE
+def increment_nonce(address):
+    nonce_object = UserNonce.query.filter_by(address=address).first()
+    if nonce_object:
+        nonce_value = nonce_object.nonce
+    else:
+        nonce_object = UserNonce(address=address)
+        nonce_value = UserNonce.FIRST_NONCE
```

Comment:
can it be assumed that nonce object exists for this address, otherwise why do increment_nonce

Reply:
I think so, but I would rather be on the safe side...